ControlRoomApplication/ControlRoomApplication/Controllers/RadioTelescopeControllers/RadioTelescopeController.cs

```diff
@@ -1116,5 +1127,20 @@ private void CheckAndRunSoftwareStops()
                 }
             }
         }
+        /// <summary>
+        /// This is the method that checks for acceptable discrepancy in the absolute and motor encoders
+        /// </summary>
+        /// <returns> True if the discrepancy is below the desired threshold for both Elevation and Azimuth values, false otherwise </returns>
+        /// <param name="motor"></param>
```

Comment:
Be sure to add some documentation for what the parameters actually mean in between the tags.

- [ ] Add parameter documentation comments